client/utils/declarations.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'wordpress-element';
```

Comment:
This file may be moved to `client/types/utils`.

Reply:
This declaration is not needed. Addressed in be848db1f8aa4e9a7d5abad5e7ae6ca0fb9c662e.